src/test/scala/com/modelfabric/sparql/stream/StreamSpec.scala

```diff
@@ -29,6 +29,7 @@ import akka.testkit.TestKit
 class StreamSpec(val _system: ActorSystem) extends TestKit(_system)
   with WordSpecLike with MustMatchers with BeforeAndAfterAll {
 
+  // JC: why this constructor is required?
```

Comment:
but this constructor is not used....
in HttpEndpointSuiteTestRunner, it's using    new StreamSpec(system),